vtex-apps/digital-river
# [DIGITALRIV-23]  Only run createCustomer when customer is logged in

**What problem is this solving?**
Don't create new customer in Digital River if customer is checking out as guest (i.e. not logged in).

**How to test it?**
Test in workspace [annadigitalriver](https://annadigitalriver--sandboxusdev.myvtex.com/).

Using an email that's not registered in Digital River (verify with `GET https://api.digitalriver.com/customers?email={email}`) place an order with [cart link](https://annadigitalriver--sandboxusdev.myvtex.com/checkout/cart/add/?sku=1111119817&qty=1&seller=1&sc=1) as a guest. Run `GET` request again and see that customer was not created.

Repeat above while logged in. Notice that this time customer was created in Digital River.
